scheduler/classes/Day.py

```diff
@@ -0,0 +1,18 @@
+import enum
+
+class DayNames(enum.Enum):
+    MONDAY = 0,
+    TUESDAY = 1,
+    WEDNESDAY = 2,
+    THURSDAY = 3,
+    FRIDAY = 4,
+    SATURDAY = 5,
+    SUNDAY = 6
+
+
+class Day:
```

Comment:
I am unsure if this should be called day, since it includes much more than the day and there is an enumeration that is similar. Maybe it should be called `DayInfo` or something along that line